src/BenchmarkDotNet/Toolchains/CsProj/CsProjGenerator.cs

```diff
@@ -19,6 +20,11 @@ namespace BenchmarkDotNet.Toolchains.CsProj
     [PublicAPI]
     public class CsProjGenerator : DotNetCliGenerator
     {
+        private const string DefaultSdkName = "Microsoft.NET.Sdk";
+
+        private static readonly ImmutableArray<string> SettingsWeWantToCopy =
+            new[] { "NetCoreAppImplicitPackageVersion", "RuntimeFrameworkVersion", "PackageTargetFallback", "LangVersion", "UseWpf" }.ToImmutableArray();
```

Comment:
You also want to copy UseWindowsForms

Reply:
@onovotny thanks!

fixed with https://github.com/dotnet/BenchmarkDotNet/commit/69a8aa51d823c6e3cf92a1fd252504d4b6d1bb3b